### DISCUSSION TITLE
error GC2F972A8: The argument type 'RawKeyEvent' can't be assigned to the parameter type 'KeyEvent'

### DISCUSSION THREAD

**Discussion by vincetphanee** [bosskmk/pluto_grid#995] [Q&A]
I had this issue even when trying with the example code. Anyone who how to fix this error ?

**Comment by darkomenx** [comment]
PlutoGrid has 36 examples of code...

**Comment by AleMeniconi18** [comment]
Suddenly I get the same error, even though weeks ago everything worked fine and I haven't changed anything

**Comment by jwalvarezuninorte** [comment]
I think it is a pluto problem with the new versions of flutter. Use a different version, for example `3.16.9` (the one I am currently using for a project using pluto grid).

For more information on how to use a different version of Flutter, consider checking out this link: https://stackoverflow.com/questions/69866692/how-to-specify-local-flutter-version-e-g-an-older-release

EDIT: Read https://github.com/bosskmk/pluto_grid/issues/968

**Comment by DevMensIT** [comment]
i also face that problem after using `pluto_grid_export` package. After removed that package its works perfectly fine.
i think something is getting conflict with `pluto_grid_export` package with `pluto_grid_plus` package
